libs/access-control/src/noops/workspace.rs

```diff
@@ -0,0 +1,60 @@
+use async_trait::async_trait;
+use uuid::Uuid;
+
+use crate::act::Action;
+use crate::workspace::WorkspaceAccessControl;
+use app_error::AppError;
+use database_entity::dto::AFRole;
+
+#[derive(Clone)]
+pub struct WorkspaceAccessControlImpl {}
+
+impl WorkspaceAccessControlImpl {
+  pub fn new() -> Self {
+    Self {}
+  }
+}
+
+impl Default for WorkspaceAccessControlImpl {
+  fn default() -> Self {
+    Self::new()
+  }
+}
+
+#[async_trait]
+impl WorkspaceAccessControl for WorkspaceAccessControlImpl {
```

Comment:
Not sure we should provide all default implement in AccessControl traits. Then we don't need to provide the default implementation its `impl`